molecule/openstack/molecule.yml

```diff
@@ -16,7 +18,7 @@ provisioner:
     name: ansible-lint
   playbooks:
     prepare: ../shared/prepare.yml
-    converge: ../shared/playbook.yml
+    converge: ../docker/playbook.yml
```

Comment:
Why did you move these out of the `shared` folder, when they're still shared?

Reply:
Moved the playbook back to shared directory and updated molecule.yml file.